README.md

```diff
@@ -51,12 +51,28 @@ graph.connect('peer1', 'peer2')
 
 > In graph theory, a directed graph (or digraph) is a graph that is a set of vertices connected by edges, where the edges have a direction associated with them.
 
-### graph = new Graph(rootElem)
+### graph = new Graph(rootElem, size)
 
 Create a new P2P graph at the root DOM element `rootElem`. In addition to an
 `Element`, a query selector string (like `'.my-cool-element'`) can also be passed
 in.
 
+Also, you can pass in an object containing the desired width and height. The possible values for both are `auto`, `default` or a number (which represents the number of pixels). Aditionally you can pass in a function that evaluates on resize.
```

Comment:
The problem here is backwards compatibility and changing it in the `webtorrent.io` website is not difficult at all, but what happens to the other users that are using this module?
I agree with you that the default and function case should disappear but I added them so each person can decide on it's own use case what will he need in his own project.